srcpkgs/trident-automount/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'trident-automount'
+pkgname=trident-automount
+version=2020.09.02
+revision=1
+wrksrc="trident-utilities-${version}"
+build_wrksrc="src-go/automount"
+makedepends="go"
+depends="autofs"
+short_desc="Automounting daemon from Project Trident utilizing autofs"
+maintainer="Ken Moore <ken@project-trident.org>"
+license="BSD-2-Clause"
+homepage="https://github.com/project-trident/trident-utilities"
+distfiles="https://github.com/project-trident/trident-utilities/archive/v${version}.tar.gz"
+checksum=0bf0991c815b56b0143106e29ff2ab952416fd63d1810a6aa51fd95a0de4c48d
+
+do_install() {
+	make
+	vbin trident-automount
+	vinstall sv/run 0755 /etc/sv/trident-automount
```

Comment:
You need to setup the `supervise` link as well.

Reply:
From what I have seen, the "supervise" link is auto-created by runit the first time the service is created. This service does not need anything special, and that auto-link works perfectly.

Reply:
`runsv` will create the `supervise` directory inside the service folder, but void uses links to `/run/runit/supervise.<service_name>` instead, because this allows runit to run with `/etc` mounted read only.

Reply:
ok, so basically just a symlink from /etc/sv/[servicename]/supervise to /run/runit/supervise.[servicename] is all I need to add?

Reply:
Yup! The `qemuconf` template does it.

Reply:
Thanks! I will get that updated here shortly.

Reply:
You should be using `vsv` instead to do all of the service copy-and-symlink work.

Reply:
`vsv` works for services in `srcpkgs/<package>/files/<service_name>`, not for package provided services.

Reply:
Good point, I overlooked that

Reply:
Ok, this is all set now in the latest version of the PR.